src/OpenLoco/src/Config.h

```diff
@@ -98,6 +98,11 @@ namespace OpenLoco::Config
         uint8_t stationNamesMinScale;                   // 0x114
         uint8_t scenarioSelectedTab;                    // 0x115
         char preferredName[256];                        // 0x116
+
+        constexpr bool hasFlags(Flags flagsToTest) const
```

Comment:
just add it to the `NewConfig` instead of the old one. You can get to the flags via `old.flags`. Eventually we will absorb the whole of old config into new config so this just simplifies one small bit.

Reply:
I made this change.